README.md

```diff
@@ -1 +1,140 @@
-# blog-app
\ No newline at end of file
+<div align="center">
```

Comment:
- [ ] **[OPTIONAL]** You have over 3000+ files changed due to the `tmp` directory. You do not have a `.gitignore` file. Kindly create a `.gitignore` file and use the default Rails settings for it.

  ```
  # See https://help.github.com/articles/ignoring-files for more about ignoring files.
  #
  # If you find yourself ignoring temporary files generated by your text editor
  # or operating system, you probably want to add a global ignore instead:
  #   git config --global core.excludesfile '~/.gitignore_global'
  
  # Ignore bundler config.
  /.bundle
  
  # Ignore all logfiles and tempfiles.
  /log/*
  /tmp/*
  !/log/.keep
  !/tmp/.keep
  
  # Ignore pidfiles, but keep the directory.
  /tmp/pids/*
  !/tmp/pids/
  !/tmp/pids/.keep
  
  # Ignore uploaded files in development.
  /storage/*
  !/storage/.keep
  .byebug_history
  
  # Ignore master key for decrypting credentials and more.
  /config/master.key
  ```